main.md

```diff
@@ -523,14 +521,16 @@ Again, the SD-JWT-R follows the same structure as the `_sd` in the SD-JWT. For E
 }
 ```
 
-The SD-JWT-R MAY contain further claims, for example, to ensure a binding
-to a concrete transaction (in the example the `nonce` and `aud` claims).
+When the holder sends SD-JWT-R to the Verifier, it MUST be a base64url-encoded 
+JWS as described in Section 5.1 of [@!RFC7515]. JWS Compact Serialization and 
+JWS JSON Serialization can be used.
```

Comment:
Are we saying somewhere why are allowing the use of the JWS JSON Serialization?  On the surface of it, this will surprise people, as JWTs only permit the use of the Compact Serialization - so as to improve interoperability.  Many JWS implementations only support the Compact Serialization.

This should be discussed in its own section possibly called "JWS Serializations Supported", rather than introduced in passing, as it is in the current PR.

Reply:
Also, see my comment https://github.com/oauthstuff/draft-selective-disclosure-jwt/issues/12#issuecomment-1153834228 .  I really don't think we should support signing formats other than the JWS Compact Serialization in the core SD-JWT spec.  If we want to define SD-JWT-like data structures using other signing formats, that should happen in different specification(s).